docs/README.md

```diff
@@ -19,3 +53,22 @@ Bake this delicious recipe and share the docs publicly with your friends with th
 2) Run `firebase deploy --only hosting:python-docs` to publish them online.
 3) Visit the [firebase hosting site](https://console.firebase.google.com/u/0/project/cord-docs/hosting/sites/python-docs), check out the public docs domain and double check if everything is to your taste.
 4) Share the docs with your friends who are most hungry for knowledge! 
+
+# TODOs / potential improvements
+- [ ] For "Pardon our dust"-sections, we could add links to (to-be-removed) docs from the web-app documentation until they are updated.
+- [ ] Datasets, Projects, Label rows, etc. are not written consistently (caps or not) across the documentation. Consider using substitutes for this.
```

Comment:
So what do you suggest here @denis-cord? 
From my point of view, there are two options here:
1. Either we just "hard code" it the right way every time 
2. or we use substitutes to make sure that they are the same.

By now, we only have four entries in `substitutes.rst` which is not a lot to keep track of. Do you think that it will be a mental overload to also include, e.g., dataset, project, and label row?